content/developers/overview/managing-deploy-keys.md

```diff
@@ -83,6 +83,32 @@ See [our guide on Git automation with tokens][git-automation].
 7. Select **Allow write access** if you want this key to have write access to the repository. A deploy key with write access lets a deployment push to the repository.
 8. Click **Add key**.
 
+##### Using multiple repositories on one server
+
+If you use multiple repositories on one server, you will need to generate a dedicated key pair for each one. You can't reuse a deploy key for multiple repositories.
+
+In the server's SSH configuration file (usually `~/.ssh/config`), add an alias entry for each repository. For example:
+
+```bash
+Host github.com-repo-0
+        Hostname github.com
+        IdentityFile=/home/user/.ssh/repo-0_deploy_key
+
+Host github.com-repo-1
+        Hostname github.com
+        IdentityFile=/home/user/.ssh/repo-1_deploy_key
+```
+
+* `Host github.com-repo-0` - The repository's alias.
+* `Hostname github.com` - Configures the alias to use the {% data variables.product.product_name %} server.
+* `IdentityFile=/home/user/.ssh/repo-0_deploy_key` - Assigns a private key to the alias.
+
+With these entries added, you can then use the alias to clone a repository, and the unique deploy key is automatically presented. For example:
```

Comment:
I think we should specifically call out that the host name needs to be replaced with the alias, and the alias is bound to the repo

```suggestion
With these entries added, you can then use the alias of the hostname to interact with the repository using SSH, so the unique deploy key for that alias used. For example:
```